collectors/debugfs.plugin/multi_metadata.yaml

```diff
@@ -100,7 +112,7 @@ modules:
                 - name: order9
                 - name: order10
             - name: mem.fragmentation_index_normal
-              description: Memory fragmentation index for each order
+              description: Memory fragmentation index for each order within the Normal zone
```

Comment:
`description` reflects the chart title

Reply:
The chart titles need updated then, they’re not properly descriptive of what each chart is.

Reply:
I can add that as a separate PR, but we should be flagging the memory zone in the chart title/description.